src/Dev/DevelopmentAdmin.php

```diff
@@ -258,4 +269,40 @@ public function errors()
     {
         $this->redirect("Debug_");
     }
+
+    public function providePermissions(): array
+    {
+        return [
+            'ALL_DEV_ADMIN' => [
+                'name' => _t(__CLASS__ . '.ALL_DEV_ADMIN_DESCRIPTION', 'Can view and run all /dev endpoints'),
+                'help' => _t(__CLASS__ . '.ALL_DEV_ADMIN_HELP', 'Can view and run all /dev endpoints'),
+                'category' => static::permissionsCategory(),
+                'sort' => 100
```

Comment:
```suggestion
                'sort' => 50
```
Doesn't have to be 50, but this should be in the list _before_ all of the other dev/* permissions so it needs a lower sort than the others.
It might also make sense to give the rest of them specifically different sort values so they're in a determined order, but I won't hold up merging if you decide not to do that. I'm not really sure what order would be best myself so I'm not fussed.